### ISSUE TITLE
BEDES unit ignored on import?

### ISSUE THREAD

**Issue by TravisWalterLBNL** [LBNL-ETA/BEDES-Manager#114]
In the import .csv file, if I specify the unit for the BEDES composite term, I don't get the same unit when I later click on that composite term in the search results (i.e., I specified "m2" in the .csv file, but the composite term had "ft2"). This seems to work properly for application terms, but not for BEDES composite terms.

**Comment by phgupta** [comment]
Per Rob, the BEDES dictionary assigned unit should be enforced ( it's okay for application term's units to be different - see issue #113 ).

**Comment by TravisWalterLBNL** [comment]
Where does the unit that actually gets used come from? From the composite term (if it already exists)? If the composite term doesn't already exist, what unit gets used? Is it using the unit from the final atomic term in the composite term?

**Comment by robHitchcock** [comment]
If the composite term exists with an assigned unit that is used.  If a new composite term rightmost atomic term has a BEDES dictionary assigned unit, that is used.  If a new composite term rightmost atomic term does not have a BEDES dictionary assigned unit, then the user may input any valid BEDES Unit Of Measure.

**Comment by TravisWalterLBNL** [comment]
I agree with that behavior, expect I wonder about making one small change: If the composite term already exists, the unit assigned to that composite term should be used, but only if that existing composite term is BEDES approved.

Are we allowing multiple copies of the same composite term if they only differ by their unit?

Any thoughts?

**Comment by robHitchcock** [comment]
I guess that additional behavior is appropriate.  I do not believe the current mapping manager code enforces any of these rules. So this all needs to be implemented.

I have only been thinking of these rules in terms of importing application mappings. But these new rules should also be consistently enforced when users manually create new composite terms.

**Comment by TravisWalterLBNL** [comment]
It seems like this behavior has changed from when I opened this issue. Was this by design, or is there a mistake somewhere?

I just imported an application term that was mapped to a new composite term whose rightmost atomic term (Area) does have a unit associated with it (ft2). In my import file, I specified m2 as the unit for the composite term. I was under the impression my m2 should have been overridden by the ft2 associated with the existing rightmost atomic term.
